### ISSUE TITLE
Does not work on Windows with non-Latin letters

### ISSUE THREAD

**Issue by TiraelSedai** [jbms/beancount-import#12]
```
Listening at http://127.0.0.1:8101
Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\beancount_import\webserver.py", line 488, in _handle_reconciler_loaded
    loaded_reconciler = loaded_future.result()
  File "C:\Python37\lib\concurrent\futures\_base.py", line 425, in result
    return self.__get_result()
  File "C:\Python37\lib\concurrent\futures\_base.py", line 384, in __get_result
    raise self._exception
  File "C:\Python37\lib\site-packages\beancount_import\thread_helpers.py", line 13, in wrapper
    f.set_result(fn(*args, **kwargs))
  File "C:\Python37\lib\site-packages\beancount_import\reconcile.py", line 380, in __init__
    self._load_sources()
  File "C:\Python37\lib\site-packages\beancount_import\reconcile.py", line 434, in _load_sources
    for spec in self.reconciler.options['data_sources']
  File "C:\Python37\lib\site-packages\beancount_import\reconcile.py", line 434, in <listcomp>
    for spec in self.reconciler.options['data_sources']
  File "C:\Python37\lib\site-packages\beancount_import\source\__init__.py", line 319, in load_source
    return m.load(source_spec, log_status=log_status)  # type: ignore
  File "C:\Python37\lib\site-packages\beancount_import\source\ofx.py", line 1373, in load
    return OfxSource(log_status=log_status, **spec)
  File "C:\Python37\lib\site-packages\beancount_import\source\ofx.py", line 1329, in __init__
    ParsedOfxFile(self.source_fitids, filename))
  File "C:\Python37\lib\site-packages\beancount_import\source\ofx.py", line 1127, in __init__
    contents = f.read()
  File "C:\Python37\lib\encodings\cp1251.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 5539: character maps to <undefined>
> c:\python37\lib\encodings\cp1251.py(23)decode()
-> return codecs.charmap_decode(input,self.errors,decoding_table)[0]
```

The journal file is encoded in UTF-8.

The workaround is to use Linux subsystem, which I used to do on my desktop, however now it's not available and my laptop doesn't have Windows 10 Pro so I'd like to fix the real issue.

P.S. Great software by the way, I really enjoyed using it in the last couple months.

**Comment by jbms** [comment]
Thanks for reporting this.  I changed beancount-import to assume all input files are UTF-8 encoded, and made a number of fixes necessary to get things working on Windows, and added a Windows Travis CI configuration to catch any regressions.  This is fixed as of: 45096bd4b4568e509f18f926305e11575f46eb4e.

**Comment by TiraelSedai** [comment]
Just tested it, works well, thanks.
